### ISSUE TITLE
links are being inserted twice

### ISSUE THREAD

**Issue by propertunist** [ColdTrick/embed_extended#6]
since i have had some issues here which appear to be not due to problems in the embed_extended code - i am thinking that my current issue is the same.. however, i don't yet know what the cause is.

i am finding that links are being inserted twice into tinyMCE and after some debugging i found that i can comment out this line in site.js and yet the item i choose to embed via the lightbox list is still embedded:
```
$(document).on('click', '.embed-item', elgg.embed_extended.insert);
```
so there must be another process that is inserting the item - hence i was seeing items being duplicated in the editor.
do you know what might be causing this? thanks

**Comment by jdalsem** [comment]
Should be fixed in v2.0.1 of this plugin https://github.com/ColdTrick/embed_extended/commit/97f5f975aa7c1260eea61bdceb84d842a863fecb

**Comment by propertunist** [comment]
oh ok, thanks - i'll come back to it when i upgrade to 2.0+ then.
